.github/workflows/sonarcloud.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         with: # At the moment the scanner requires dotnet 5 https://www.nuget.org/packages/dotnet-sonarscanner
           dotnet-version: |
             5.x
```

Comment:
I don't think SQ needs net5 anymore, we can probably drop it at the same occasion